code/cryptography/src/atbash_cipher/atbash_cipher.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
```

Comment:
Be sure to look at the java style guide, and refactor your code to make it more readable.

Reply:
Sure! I'll do that. Thanks for suggestions.

Reply:
You should add empty lines between the different parts of your code.